pkg/proxy/proxy.go

```diff
@@ -50,18 +51,90 @@ func NewProxy(maxLoadTestsRun int, kubeClient *kube.Client, specCreator loadTest
 
 // LoadTestStatus defines response structure for status request
 type LoadTestStatus struct {
-	Type            string `json:"type"`
-	DistributedPods int32  `json:"distributedPods"`        // number of distributed pods requested
-	Namespace       string `json:"loadtestName,omitempty"` // namespace created equals the loadtest name
-	Phase           string `json:"phase,omitempty"`        // jmeter loadtest status
-	HasEnvVars      bool   `json:"hasEnvVars"`
-	HasTestData     bool   `json:"hasTestData"`
+	Type            string            `json:"type"`
+	DistributedPods int32             `json:"distributedPods"`        // number of distributed pods requested
+	Namespace       string            `json:"loadtestName,omitempty"` // namespace created equals the loadtest name
+	Phase           string            `json:"phase,omitempty"`        // jmeter loadtest status
+	Tags            map[string]string `json:"tags"`
+	HasEnvVars      bool              `json:"hasEnvVars"`
+	HasTestData     bool              `json:"hasTestData"`
 }
 
 func getLoadTestType(r *http.Request) apisLoadTestV1.LoadTestType {
 	return apisLoadTestV1.LoadTestType(r.FormValue(backendType))
 }
 
+// List lists all the load tests.
+func (p *Proxy) List(w http.ResponseWriter, r *http.Request) {
+	logger := mPkg.GetLogger(r.Context())
+
+	ctx, cancel := context.WithTimeout(r.Context(), loadtest.KubeTimeout)
+	defer cancel()
+
+	opt := kube.ListOptions{}
```

Comment:
I think we should introduce pagination here to avoid a possibly overloading the service if there are a lot of load tests.
See Limit and Continue under [ListOptions](https://godoc.org/k8s.io/apimachinery/pkg/apis/meta/v1#ListOptions)